test/utils/shippable/sanity.sh

```diff
@@ -1,30 +1,44 @@
 #!/bin/bash -eux
 
 source_root=$(python -c "from os import path; print(path.abspath(path.join(path.dirname('$0'), '../../..')))")
+ansible_repo_url="https://github.com/ansible/ansible.git"
```

Comment:
Move this inside the repomerge block below since it is only used there. Also, since it's only used once it can be inline in the clone command.